__tests__/app.test.js

```diff
@@ -190,6 +190,25 @@ describe("/api/articles", () => {
         });
       });
   });
+  test("GET 200: responds with an array of articles filtered by the passed topic query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({body}) => {
+        const {articles} = body;
+        expect(articles.every((article) => article.topic === "mitch")).toBe(
+          true
+        );
```

Comment:
This would produce a false positive if articles is empty (because an empty array is truthy!) so therefore a length check is needed beforehand :)